Fix parsing values in text protocol containing spaces or newlines

imrivera opened the pull request:
Hi!

I've updated the text protocol worker, so it correctly parses values containing spaces or newlines. I also added a SUITE for this module following the structure of the binary one.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/AdRoll/mero?pullRequest=46) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/AdRoll/mero?pullRequest=46) before we can accept your contribution.<br/><hr/>**Ignacio Martínez** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/AdRoll/mero?pullRequest=46) it.</sub>

miriampena commented:
👋

imrivera commented:
:wave: Hi Miriam!

Great work in this app. We are going to start using it :)

imrivera commented:
Sorry, I forced a push in my branch, I rebased the changes on your latest master commit. I also removed a couple of `binary:copy` calls.

Do you think this could be merged into your master?

elbrujohalcon commented on Makefile:
```diff
@@ -0,0 +1,38 @@
+REBAR=rebar
```
Please don't re-add a `Makefile`. Particularly not one using the old `rebar`. We use `rebar3` directly to maintain this project now.

elbrujohalcon commented on test/mero_SUITE.erl:
```diff
@@ -332,16 +332,20 @@ multiget_defineds(_Conf) ->
-                {<<"16">>, <<"3">>},
-                {<<"12">>, <<"1">>},
-                {<<"11">>, <<"1">>}],
+    Expected = lists:sort([{<<"17">>, undefined},
```
You could've just sorted the list in place, instead of adding a `lists:sort/1` call ;)

elbrujohalcon commented on test/mero_SUITE.erl:
```diff
@@ -353,17 +357,18 @@ multiget_defineds_clustered_keys(_Conf) ->
     %% 13, 14 and 15 will go to the same server
     %% 11, 12 and 16 to a different one
-    Expected = [{<<"17">>, undefined},
+    Expected = lists:sort(
```
Same as above

elbrujohalcon commented on test/mero_wrk_tcp_txt_SUITE.erl:
```diff
@@ -0,0 +1,233 @@
+        FakeNetwork = spawn_link(fun() -> fake_network_recv(Buffer, ReadSize) end),
+        meck:expect(gen_tcp, recv, fun(_, 0, _Timeout) -> network_read(FakeNetwork) end),
+        {ok, Client} = mero_wrk_tcp_txt:connect("localhost", 5000, {?MODULE, stats, []}),
+        %dbg:tracer(),
```
I don't think we need to keep this debug calls on the test.

imrivera commented:
Thank you for the feedback. To avoid cluttering even more this pull-request I'm going to close this one and open a clean new one
